src/openloco/window.cpp

```diff
@@ -604,6 +604,16 @@ namespace openloco::ui
         }
     }
 
+    bool window::moveToCentre()
+    {
+        float dx = ((ui::width() - this->width) / 2);
+        float dy = ((ui::height() - this->height) / 2);
```

Comment:
No need to use floats here; in fact, decimal precision is not desirable for pixels here. Note that window::move also takes two int16_t as its parameters.
```suggestion
        int16_t dx = ((ui::width() - this->width) / 2);
        int16_t dy = ((ui::height() - this->height) / 2);
```